addons/test_mail/tests/test_message_post.py

```diff
@@ -929,7 +929,12 @@ def test_message_post_w_attachments(self):
         )
         _attachment_records[1].write({'mimetype': 'image/png'})  # to test message_main_attachment heuristic
 
-        test_record = self.env['mail.test.simple'].browse(self.test_record.ids)
+        test_record = self.env['mail.test.simple.main.attachment'].with_context(self._test_context).create({
```

Comment:
Would update test function name to hilight we are testing the main attachment mixin. We can also add a small docstring while we are at it :)

Reply:
As main attachment does the base job + some additional job, we can probably consider code is sufficiently tested.